Updating docs (#279)
NREL/BuildingMOTIF: docs/tutorials/model_correction.md

# print some of the autogenerated template
for templ in generated_templates.get_templates():
    templ = templ.inline_dependencies()

>     # skipping uninteresting templates for the sake of the demo

Reviewer: I understand what's happening here, but I wonder if there's a better way to get the same result in this tutorial. I don't want this line to start showing up in people's code who have just copy pasted and not read the comment.

    if len(templ.parameters) < 4:
        continue
    print(f"Name (autogenerated): {templ.name}")
    print(f"Parameters (autogenerated): {templ.parameters}")
    print("Template body (autogenerated):")
    print(templ.body.serialize())
    print('-' * 79)
```

In this case, the generated templates are fairly simple. They require an input for the name of the supply fan and the names of several missing points. We can loop through each of these generated templates and create the names. Here, we are creating arbitrary names for the points but in a real setting you would likely pull the equipment or point names from an external source like a Building Information Model or BACnet network[^1] (see future tutorials for how to do this!) Another challenge is the fact that we already have a supply fan in the model. Here, we can take advantage of the fact that the name of the fan in the existing model are just the name of the AHU wtih the `-Fan` postfix. The name of the AHU is in the generated templates (see above) so we can just pull out the name of the AHU, add the postfix, and use that as the value for the `name` parameter.

> If we just add the generated templates to the building model, we will probably pass validation *but* the entities will not have names with any significance to the building. It is highly recommended to use the template evaluation features (demonstrated below) to fill in the parameters with the "real" names of the entities as they appear in the building and/or building management system.

Reviewer: ```suggestion
If we just add the generated templates to the building model, we will probably pass validation *but* the entity names will have no significance to the building. It is highly recommended to use the template evaluation features (demonstrated below) to fill in the parameters with the "real" names of the entities as they appear in the building and/or building management system.
```
